erddapy/core/interfaces.py

```diff
@@ -8,26 +8,29 @@
 import iris
 import pandas as pd
 import xarray as xr
-from netCDF4 import Dataset as ncDataset
+from netCDF4 import Dataset
 
 from erddapy.core.netcdf import _nc_dataset, _tempnc
 from erddapy.core.url import urlopen
 
 
-def to_pandas(url: str, requests_kwargs=dict(), **kw) -> pd.DataFrame:
+def to_pandas(url: str, requests_kwargs=None, **kw) -> pd.DataFrame:
     """Convert a URL to Pandas DataFrame."""
+    if requests_kwargs is None:
+        requests_kwargs = dict()
```

Comment:
It is kind of standard to initialize dicts like this. I know, less readable, but there are some optimizations at play here.

```suggestion
        requests_kwargs = {}
```